UnityProject/Assets/Scripts/PlayGroups/Input/InputController.cs

```diff
@@ -54,6 +54,11 @@ private void Update()
 			CheckHandSwitch();
 			CheckClick();
 			CheckAltClick();
+			// Useful for debugging death and ghost related issues
+			//if(Debug.isDebugBuild)

```

Comment:
Why comment this out if you already have the "isDebugBuild" check on it?

Reply:
In case someone is debugging the chat. This gets uncommented if you are specifically debugging death related stuff.

Reply:
Things dont magically get uncommented ;)

I do get your point, but maybe you should just remove this shit and do it like Doobly advised.
We prefer solutions that are actually usefull in the final product. If you can add /suicide trigger in chat, that is prefered to this solution.

Just remove this and get your actually requested fix in only. You can later PR a good suicide if you want to ;)

Reply:
Should I leave the PNA commands in, and just remove the input check?

Reply:
That would be fine with me 👍